### DISCUSSION TITLE
How to Disable Landing Page

### DISCUSSION THREAD

**Discussion by iqlal** [vuejs/vitepress#2437] [Q&A] [answered]
Hi, I want to ask, "how to disable landing page in vitepress an straight to the docs page?"

**Comment by kiaking** [comment, marked as answer]
Don't use frontmatter `layout: home` then all pages becomes docs page.

Are you looking for something like this site?
https://sefirot.globalbrains.com

Then the config for this is at here 👇 
https://github.com/globalbrain/sefirot/blob/main/docs/.vitepress/config.ts

**Reply by iqlal** [reply]
Thanks :)
